laterpay/__init__.py

```diff
@@ -302,19 +308,38 @@ def _get_web_url(self,
 
     def get_buy_url(self, item_definition, *args, **kwargs):
         """
-        Get the URL at which a user can start the checkout process to buy a single item.
+        Get the URL at which a user can start the checkout process to buy a
+        single item or contribute/donate to a campaign.
 
         http://docs.laterpay.net/platform/dialogs/buy/
         """
-        return self._get_web_url(item_definition, 'buy', *args, **kwargs)
+        page_type = 'buy'
+        item_type = item_definition.item_type
+        if not item_type:
```

Comment:
Not a blocker, but this seems a bit odd. What if you omitted this?

Reply:
My first intention was to default to the first if branch for faster execution, but getting coverage right was going to be tricky. Also, premature optimization.